shark-graph/src/main/java/shark/internal/hppc/LongLongScatterMap.kt

```diff
@@ -186,7 +191,7 @@ class LongLongScatterMap {
         while (slot < max) {
           existing = keys[slot]
           if (existing != 0L) {
-            block(existing, values[slot])
+            forEachCallback.onEntry(existing, values[slot])
```

Comment:
Because the lambda was a `Function2<Long, Long>` this caused a call to `Long.valueOf`